ci/travis/build-docker-images.py

```diff
@@ -0,0 +1,230 @@
+import datetime
+import glob
+import os
+import re
+import shutil
+import subprocess
+import sys
+from typing import List
+
+import docker
+
+DOCKER_USERNAME = "raytravisbot"
+DOCKER_CLIENT = None
+
+
+def _subprocess_wrapper(cmd):
+    result = subprocess.run(
+        cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+    if result.returncode != 0:
+        print(f"COMMAND FAILED: {cmd}")
+        print(f"STDOUT is: {result.stdout}")
+        print(f"STDERR is: {result.stderr}")
+        raise Exception("Command Failed") from None
+    return (result.stdout or b"").decode()
+
+
+def _merge_build():
+    return os.environ.get("TRAVIS_PULL_REQUEST") == "false"
+
+
+def _release_build():
+    branch = os.environ.get("$TRAVIS_BRANCH")
+    if not branch:
+        print("Branch not found!")
+        return False
+    return branch != "master" and "releases" in branch
+
+
+def _get_curr_dir():
+    return os.path.dirname(os.path.realpath(__file__))
+
+
+def _get_root_dir():
+    return os.path.join(_get_curr_dir(), "../../")
+
+
+def _get_wheel_name():
+    matches = glob.glob(f"{_get_root_dir()}/.whl/*cp37m-manylinux*")
```

Comment:
maybe move the 37 up to a global variable so it's easy to change version later on?